docs/releases/202402.mdx

```diff
@@ -4,6 +4,15 @@ title: February 2024
 displayed_sidebar: releaseSideBar
 ---
 
+### February 8 2024
+**Improvements**
+- Node name is now displayed in the editor notebook browser tab.
+- The Benchling class of the Ganymede SDK will now retry to register entities when it encounters an error, and will conditionally error when an empty assay result is returned.
+
+**Bug Fixes**
+- Flow Run param return values were corrected. Previously, multi-param nodes returned no more than 2 params.
```

Comment:
this is neat - not clear what this means

Reply:
This is [the ticket ](https://ganymede-squad.monday.com/boards/3098598431/pulses/5975117462)- essentially, UDC would only see two params from a multi-param node. Seems like it didn't cause any issues, so I could maybe remove.